Cargo.toml

```diff
@@ -30,10 +30,10 @@ pre-release-replacements = [
 ]
 
 [dependencies]
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "1.0.113", features = ["derive"] }
 serde_json = "1.0"
 once_cell = "1.2.0"
-log = "0.4"
+log = "0.4.4"
```

Comment:
If you want to break this out into its own PR, I'm willing to merge it without waiting on the other parts